weather_aplication/SearchViewController.swift

```diff
@@ -0,0 +1,130 @@
+import SnapKit
+import UIKit
+
+struct Location {
+    public var city: String?
+}
+
+class SearchViewController: UIViewController {
+    private let kSideMargin: CGFloat = 10
+    private let kTopMargin: CGFloat = 20
+    private let kBottomMargin: CGFloat = 15
+    
+    private let cityNameTextfield = UITextField()
+    private let cityNamesTableView = UITableView()
+    var cityNameArray: [String] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupConstraints()
+        cityNameTextfield.addTarget(self, action: #selector(loadCityNames), for: .editingChanged)
+        
+    }
+    
+    @objc func loadCityNames() {
+        guard let cityName = cityNameTextfield.text else {return}
+        getCityName(fromCity: cityName, completion: {(location) -> Void in
+            guard let results = location else {return}
+            if !results.isEmpty {
+                var cityArray: [String] = []
+                for result in results{
+                    guard let city = result.city else {break}
+                    cityArray.append(city)
+                }
+                self.cityNameArray = cityArray
+            } else {
+                return
+            }
+          })
+        cityNamesTableView.reloadData()
```

Comment:
To jest umieszczone w złym miejscu. Pomyśl i napisz mi gdzie i dlaczego to tam powinno być